lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -711,6 +710,15 @@ spec_balanceTransaction = describe "balanceTransaction" $ do
                 `shouldBe`
                 Left (ErrBalanceTxAssignRedeemers
                         (ErrAssignRedeemersTargetNotFound faultyRedeemer))
+
+    describe "merging and splitting positive and negative parts of values" $ do
+
+        it "prop_mergeValuePositiveNegative_splitValuePositiveNegative" $
+            prop_mergeValuePositiveNegative_splitValuePositiveNegative
+                & property
+        it "prop_splitValuePositiveNegative_mergeValuePositiveNegative" $
+            prop_splitValuePositiveNegative_mergeValuePositiveNegative
+                & property
```

Comment:
I like having properties have readable descriptions in the hspec tree. What do you think about:
```suggestion
    describe "splitValuePositiveNegative and mergeValuePositiveNegative" $ do

        it "merge . split === id" $
            prop_mergeValuePositiveNegative_splitValuePositiveNegative
                & property
        it "split . merge === id (if disjoint)" $
            prop_splitValuePositiveNegative_mergeValuePositiveNegative
                & property
```
or some variant of it?

Reply:
> I like having properties have readable descriptions in the hspec tree. What do you think about ...

My main objections to repeating the property definition in the hspec description:

1. It violates the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself):
    - If we update the property and forget to update the string description, the string will then be erroneous.
    - The string description is not checked by the compiler.
2. It requires the property definition to fit on a single line (or part of a single line):
    - But some properties can only be easily written down on multiple lines.
    - How should we format these?
3. It becomes more awkward to match and run just a single property test.
    - By using the function name as a label, we can guarantee that the label is unique.